### ISSUE TITLE
Currently selected file always gets dragged when scrolling the file tree

### ISSUE THREAD

**Issue by BuriedInTheGround** [lockbook/lockbook#3068]
On Linux. This often leads to inadvertently moving a document into another folder.

https://github.com/user-attachments/assets/a2ee040f-28b8-4214-b677-904ae3ac5daa

**Comment by tvanderstad** [comment]
Is that what your cursor icon usually looks like? Seems like cursor icons are broken for you

**Comment by BuriedInTheGround** [comment]
Yes, that's normal. I never bothered installing a cursor pack since I don't use the mouse that much, and inside applications that don't explicitly ask for the cursor type they want (e.g. most non-browser apps) that "X" cursor is somehow the default fallback.

**Comment by Parth** [comment]
@tvanderstad closed in https://github.com/lockbook/lockbook/pull/3257 or https://github.com/lockbook/lockbook/pull/3229?

**Comment by tvanderstad** [comment]
> [@tvanderstad](https://github.com/tvanderstad) closed in [#3257](https://github.com/lockbook/lockbook/pull/3257) or [#3229](https://github.com/lockbook/lockbook/pull/3229)?

Yeah, in #3229, thanks!
